src/DssSpell.sol

```diff
@@ -65,82 +45,24 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-        // https://vote.makerdao.com/polling/QmVkRdjg#poll-detail
-        bytes32 _ilk  = bytes32("TUSD-A");
-        address _clip = DssExecLib.getChangelogAddress("MCD_CLIP_TUSD_A");
-        //
-        // Enable liquidations for TUSD-A
-        // Note: ClipperMom cannot circuit-break on a DS-Value but we're adding
-        //       the rely for consistency with other collaterals and in case the PIP
-        //       changes to an OSM.
-        DssExecLib.authorize(_clip, DssExecLib.clipperMom());
-        DssExecLib.setValue(_clip, "stopped", 0);
-        // Use Abacus/LinearDecrease
-        DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_TUSD_A);
-        // Set Liquidation Penalty to 0
-        DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
-        // Set Liquidation Ratio to 150%
-        DssExecLib.setIlkLiquidationRatio(_ilk, 15000);
-        // Set Auction Price Multiplier (buf) to 1
-        DssExecLib.setStartingPriceMultiplicativeFactor(_ilk, 10000);
-        // Set Local Liquidation Limit (ilk.hole) to 5 million DAI
-        DssExecLib.setIlkMaxLiquidationAmount(_ilk, 5 * MILLION);
-        // Set tau for Abacus/LinearDecrease to 21,600,000 second (estimated 10bps drop per 6 hours = 250 days till 0)
-        DssExecLib.setLinearDecrease(MCD_CLIP_CALC_TUSD_A, 21_600_000);
-        // Set Max Auction Duration (tail) to 432,000 seconds (5 days, implies minimum price of 0.98)
-        DssExecLib.setAuctionTimeBeforeReset(_ilk, 432_000);
-        // Inferred by risk. Enforcing here with approval https://discord.com/channels/893112320329396265/897483518316265553/963575806653767720
-        DssExecLib.setAuctionPermittedDrop(_ilk, 9800);
-        // Set Proportional Kick Incentive (chip) to 0
-        DssExecLib.setKeeperIncentivePercent(_ilk, 0);
-        // Set Flat Kick Incentive (tip) to 500
-        // NOTE: PECU rejected the incentive update to 500.
-        //       https://forum.makerdao.com/t/tusd-offloading-flat-kick-incentive-change/14645
-        DssExecLib.setKeeperIncentiveFlatRate(_ilk, 0);
-        // Update spotter price
-        DssExecLib.updateCollateralPrice(_ilk);
-        // Update calc in changelog
-        DssExecLib.setChangelogAddress("MCD_CLIP_CALC_TUSD_A", MCD_CLIP_CALC_TUSD_A);
-
-        // Update changelog version
-        DssExecLib.setChangelogVersion("1.11.1");
+        // --------------------------------- Oasis.app OSM Whitelist ---------------------------------------
+        // https://vote.makerdao.com/polling/QmZykRSM
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_ETH"),    OASIS_APP_OSM_READER);
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_WSTETH"), OASIS_APP_OSM_READER);
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_WBTC"),   OASIS_APP_OSM_READER);
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_RENBTC"), OASIS_APP_OSM_READER);
```

Comment:
It is the same address than the WBTC Osm.

Reply:
I've commented the line in c2dcd55, keeping it in the tests to ensure coverage.